front/lib/api/assistant/actions/tables_query.ts

```diff
@@ -144,27 +144,56 @@ async function tablesQueryActionSpecification({
       {
         name: "question",
         description:
-          "The plain language question to answer based on the user request and conversation context. The question should include all the context required to be understood without reference to the conversation. If the user has multiple unanswered questions, make sure to include all of them. If the user asked to correct a previous attempt at the same query in a specific way, this information must be included.",
+          "The plain language question to answer based on the user" +
+          " request and conversation context. The question should include" +
+          " all the context required to be understood without reference to the conversation." +
+          " If the user has multiple unanswered questions, make sure to include all of them." +
+          " If the user asked to correct a previous attempt at the same query in a specific way," +
+          " this information must be included.",
         type: "string" as const,
       },
     ],
   };
 }
 
 // Generates the action specification for generation of rawInputs passed to `runTablesQuery`.
+export async function deprecatedGenerateTablesQuerySpecificationForSingleActionAgent(
+  auth: Authenticator
+): Promise<Result<AgentActionSpecification, Error>> {
+  const owner = auth.workspace();
+  if (!owner) {
+    throw new Error("Unexpected unauthenticated call to `runQueryTables`");
+  }
+
+  const actionDescription =
+    "Query the structured data tables specificied by the user to retrieve information to answer their request." +
```

Comment:
`Query the data tables specified by the user by executing a generated SQL query from the natural language.`